rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -126,14 +126,16 @@ def add(self, episodes: Union[List["SingleAgentEpisode"], "SingleAgentEpisode"])
                 eps_idx = self.episode_id_to_index[eps.id_]
                 existing_eps = self.episodes[eps_idx - self._num_episodes_evicted]
                 old_len = len(existing_eps)
-                self._indices.extend([(eps_idx, old_len + i) for i in range(len(eps))])
+                self._indices.extend(
```

Comment:
Question: Doesn't this additional None lead to an error when we unpack/interpret `self._indices` somewhere else in this class?

Reply:
It does - I might have accidentially added this there while it should have gone into the prioritized buffer. Good catch!